### ISSUE TITLE
Version 0.6.0 not aviable on plugins.gradle.org

### ISSUE THREAD

**Issue by simartn** [ksoichiro/gradle-console-reporter#13]
I tried to upgrade to version 0.6.0 which should be compatible with gradle 5.x but failed. Is there another maven repository where I can download the plugin?

## Steps to reproduce

```sh
mkdir test

cd test

gradle wrapper

echo 'plugins {
    id "com.github.ksoichrio.console.reporter" version "0.6.0"
}
' > build.gradle

./gradlew test

FAILURE: Build failed with an exception.

* Where:
Build file '/Users/emundo/Documents/Projekt/BMW/PPT/Code/test/build.gradle' line: 2

* What went wrong:
Plugin [id: 'com.github.ksoichrio.console.reporter', version: '0.6.0'] was not found in any of the following sources:

- Gradle Core Plugins (plugin is not in 'org.gradle' namespace)
- Plugin Repositories (could not resolve plugin artifact 'com.github.ksoichrio.console.reporter:com.github.ksoichrio.console.reporter.gradle.plugin:0.6.0')
  Searched in the following repositories:
    Gradle Central Plugin Repository

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

**Comment by ksoichiro** [comment]
I've already published.
https://plugins.gradle.org/plugin/com.github.ksoichiro.console.reporter

Maybe it's because your spelling is wrong.
`ksoichrio` should be `ksoichiro`.

Please copy and paste this:

```groovy
plugins {
  id "com.github.ksoichiro.console.reporter" version "0.6.0"
}
```

**Comment by mneundorfer** [comment]
I can second that. The spelling is correct, but i does not work. `0.5.0` works just fine

**Comment by simartn** [comment]
The typo is correct, but with the correct ID it does not work either. I see it published on the website but I am not able to download it. Thank you for your reply!

**Comment by yq314** [comment]
It looks like it's not published correctly, if you click on any link on this page https://plugins.gradle.org/m2/gradle/plugin/com/github/ksoichiro/gradle-console-reporter/0.6.0/ they redirect to an error page saying specified key doesn't exist.

https://plugins.gradle.org/m2/gradle/plugin/com/github/ksoichiro/gradle-console-reporter/0.5.0/ just works fine.

**Comment by ksoichiro** [comment]
@yq314 Thanks, I confirmed it.
@simonnagl @mneundorfer Sorry for my misunderstanding about the situation.
I'll retry publishing it later.

**Comment by ksoichiro** [comment]
I've just released v0.6.2.
Maybe you can use it now.

(v0.6.0 has been already published and couldn't be overridden.
I tried again as v0.6.1, which was the same result.)
Then I updated the Gradle publish plugin and it seems to be successfully uploaded.
Maybe the publish plugin was too old.

**Comment by simartn** [comment]
version `0.6.2` can be downloaded. Thank you!
